components/calendar--event.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <li
+    v-popover="{
+      name: firstEvent.id
+    }"
+    class="
+      block list-none mx-1 py-2 border-t border-blue
+      hover:bg-blue-lightest
+    "
+  >
+    <a
+      :href="firstEvent.url"
+      class="w-full block no-underline text-blue-darker"
+      target="_blank"
+      rel="noreferrer noopener"
+    >
+      <span v-if="eventGroup(firstEvent)">
+        <logo
+          v-if="eventGroup(firstEvent)"
+          :icon-set="eventGroup(firstEvent).iconSet"
+          :icon-name="eventGroup(firstEvent).iconName"
+          :icon-text="eventGroup(firstEvent).iconText"
+          class="inline-block"
+        />
+      </span>
+      {{ formatTimeOfEvent(firstEvent) }}
+      <div class="mt-1">{{ eventGroup(firstEvent).name }}</div>
+      <div
+        v-if="additionalEvents.length > 0"
+        class="text-sm">
+        <i>also:
+          <span
+            v-for="(event, index) in additionalEvents"
+            :key="event.id">
+            <span v-if="index == 0">
+              {{ formatTimeOfEvent(event) }}
+            </span>
+            <span v-else>
+              , {{ formatTimeOfEvent(event) }}
+            </span>
```

Comment:
Can you remove the whitespace between the `span` element and the content? It's leading to a space appearing before the comma on the page:

![image](https://user-images.githubusercontent.com/5572255/67597535-746ce180-f739-11e9-80c8-4b6b39162a0b.png)

I think one of these would do the trick, but I'm not sure which one will appease the linter:

``` html
<span v-else>, {{ formatTimeOfEvent(event) }}</span>
```

or

``` html
<span
  v-else
>, {{ formatTimeOfEvent(event) }}</span>
```

Reply:
Thanks. I didn't know how to test this. Maybe you can show me sometime but I'll remove the space as you suggested.